src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -140,7 +140,6 @@ private RuleSetInformation[] GetAggregatedSolutionRuleSets(BoundSonarQubeProject
                 string baselineRuleSet = ruleSetInfoProvider.CalculateSolutionSonarQubeRuleSetFilePath(
                     bindingInfo.ProjectKey,
                     Language.ForProject(project),
-                    // TODO: duncanp decide whether to support new connected mode
```

Comment:
I decided not to enable the rule-conflict checking for the new connected mode (i.e. the complicated code that checks whether the user has modified the ruleset to weaken it), many due to time constraints, but also because I wasn't certain that this was a behaviour we wanted to keep. If necessary, we can add it later.